### DISCUSSION TITLE
AppVeyor blocks my PR, what to do?

### DISCUSSION THREAD

**Discussion by Timwi** [gitextensions/gitextensions#9605] [Q&A] [answered]
Hi. So I managed to clone, compile and run Git Extensions. I made a super trivial minor change to it and [submitted a pull request](https://github.com/gitextensions/gitextensions/pull/9603).

The pull request shows AppVeyor reporting a build failure. It builds fine for me, and it’s extremely far-fetched to think that my changes would cause the build failure. Any help?

**Comment by RussKie** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/4403806/135580467-f0a6fff0-1702-419d-8087-17d5180d91b7.png)

**Comment by pmiossec** [comment]
@Timwi It's easier to have everything in the same place. So, in the future, better add a comment in the PR that failed. I have answered there. And in the mean time, @RussKie answered also ;)

**Reply by Timwi** [reply]
Thank you so much! You guys are very helpful.
